docs/faq-build.md

```diff
@@ -0,0 +1,31 @@
+Support for moving from development to production with SAFE applications is included "out of the box" when using e.g. [Azure](template-appservice) or [Docker](template-docker) deployment options. Nonetheless, this page explains the key differences that you should be aware of between running SAFE applications in development and production.
+
+## Developing SAFE applications
+The SAFE template is geared up towards a streamlined development process. Firstly, it contains a [FAKE build script](template-fake) which you can use this to quickly download dependencies, as well as build and run both the client and server on your machine.
+
+The development process uses *two web servers* running in tandem:
+
+* The "back-end" .NET web server which hosts your API endpoints and other server-generated content.
+    * For **[Saturn](component-saturn)** or **Giraffe**, this uses Kestrel on ASP .NET out of the box (although you can opt for [other options]()),
+    * **Suave** uses its own built-in proprietary HTTP server.
```

Comment:
I guess it's just my bad English - but in context of software `proprietary` tends to be understand as closed-source/paid software.

Reply:
Good point, I'll change it to something clearer.